### ISSUE TITLE
[TW-351] A couple of spelling corrections for taskrc man page

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#386]
_Ben Armstrong on 2013-08-01T15:53:22Z says:_ 

 I found a couple of spelling mistakes and corrected them:

https://github.com/taskwarrior/task/pull/2

This is my first bug. If there's anything I should change about workflow, please let me know. :)

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2013-08-01T15:53:22Z
Modified: 2014-02-09T01:36:03Z
```

**Comment by taskwarrior** [comment]
_Scott Kostyshak on 2013-08-02T01:43:10Z says:_ 

 Hi Ben,

Thanks! I commited the fixes at 38bba720. As far as patch suggestions, see here:
http://taskwarrior.org/wiki/1/What_makes_a_good_patch

I don't think we actively use the github repo.
